commands.py

```diff
@@ -500,19 +500,19 @@ def get_settings(self):
     def setting_was_changed(self, setting):
         """Update all views when the lint mode changes."""
         if setting == 'background':
-            from .sublimelinter import SublimeLinter
+            from .sublime_linter import SublimeLinter
             SublimeLinter.lint_all_views()
         else:
             linter.Linter.clear_all()
 
 
-@choose_setting_command('mark_style', preview=True)
-class SublimelinterChooseMarkStyleCommand(ChooseSettingCommand):
-    """A command that selects a mark style from a list."""
+# @choose_setting_command('mark_style', preview=True)
```

Comment:
We can no longer support commands like this. Please also remove the related menu entries, and check if there are docs about it.